src/components/list.js

```diff
@@ -3,6 +3,51 @@ import PropTypes from 'prop-types';
 import { getStyles } from '../utils/base';
 import styled from 'react-emotion';
 
+// Is it okay to export from here?  Should they go in different files?
+export const bulletStyles = {
+  star: '\\2605',
+  classicCheck: '\\2713',
+  greenCheck: '\\2705',
+  arrow: '\\219d',
+  cross: '\\274C'
+};
+
+export const getBulletStyle = (bulletStyle, isListItemStyle) => {
```

Comment:
@ebrillhart I believe this whole piece of styling should go to the body.js file, however, I was not able to move it with Mariano's help as this file required `getBulletStyle ` function to return an object which is not an option at this moment as we are returning an object and a string at once. Is there any built in method I can use to convert it to an object or it is ok to keep it here for now?

Reply:
we can keep it like this for now!